package.json

```diff
@@ -74,18 +77,22 @@
     "prettier": "2.7.1",
     "sass": "^1.28.0",
     "sass-loader": "^7.1.0",
-    "storybook-addon-designs": "^6.3.1",
+    "storybook": "^8.3.4",
     "stylelint": "13.8.0",
     "stylelint-config-standard": "20.0.0",
     "stylelint-order": "6.0.3",
     "stylelint-scss": "3.18.0",
     "ts-jest": "29.0.5",
     "ts-node": "^10.9.1",
     "typescript": "4.6.4",
+    "vite": "^5.4.7",
+    "vite-plugin-checker": "^0.7.0",
+    "vite-plugin-css-injected-by-js": "^3.5.1",
+    "vite-plugin-dts": "^3.9.1",
+    "vite-svg-loader": "^5.1.0",
     "vue": "3.2.45",
     "vue-loader": "17.0.0",
```

Comment:
I think we can remove it together with `@vue/cli-*`, `sass-loader`, `vite.config.js` and maybe something more.